### ISSUE TITLE
Console warning with Select component on Expo web - Unknown event handler property onHoverIn

### ISSUE THREAD

**Issue by brianGammon** [gluestack/gluestack-ui#1779]
### Description

When rendering the `<Select>` component on web from an Expo 50 project a warning appears in the browser console: `Unknown event handler property 'onHoverIn'. It will be ignored.`

There is a second warning for 'onHoverOut' as well.

### Steps to reproduce

1. Create a new Expo project using `npm create gluestack`
2. Edit the App.tsx page and place the sample select components from the GS docs:
```
<Select>
    <SelectTrigger variant="outline" size="md">
      <SelectInput placeholder="Select option" />
      <SelectIcon as={ChevronDownIcon} />
    </SelectTrigger>
    <SelectPortal>
      <SelectBackdrop />
      <SelectContent>
        <SelectDragIndicatorWrapper>
          <SelectDragIndicator />
        </SelectDragIndicatorWrapper>
        <SelectItem label="UX Research" value="ux" />
        <SelectItem label="Web Development" value="web" />
        <SelectItem
          label="Cross Platform Development Process"
          value="Cross Platform Development Process"
        />
        <SelectItem
          label="UI Designing"
          value="ui"
          isDisabled={true}
        />
        <SelectItem label="Backend Development" value="backend" />
      </SelectContent>
    </SelectPortal>
  </Select>
  ```
3. Run `npm run web`
4. Open the browser console and refresh the page
5. See the warning: Unknown event handler property `onHoverIn`. It will be ignored.

### gluestack-ui Version

1.1.4

### Platform

- [X] Expo
- [ ] React Native CLI
- [ ] Next
- [ ] Web
- [ ] Android
- [ ] iOS

**Comment by jezzdk** [comment]
I see it too. It still seems to function on web but it is not working at all in iOS.

**Comment by brianGammon** [comment]
It's working for me on iOS. If you are rendering Select on a modal screen, using `presentation: 'modal'` be sure to add the useRNModal property (not well documented).

**Comment by RishabhSahu24** [comment]
@brianGammon 
Thank you for reporting the issue. We have addressed it and a pull request (PR) has been submitted.

**Comment by brianGammon** [comment]
@RishabhSahu24 - Thanks for fixing these warnings. I am seeing a regression on the Select component with the updated version. On 1.1.4, the selected item was highlighted on the ActionSheet (on native), however now on 1.1.6 the selected item is not highlighted at all. Screenshots before and after:

GLuestack 1.1.4
<img width="250" alt="iPhone_14-3" src="https://github.com/gluestack/gluestack-ui/assets/16623517/b7859748-6e0c-41e0-9a1d-ffbb366f58a4">

Gluestack 1.1.6
<img width="250" alt="iPhone_14-4" src="https://github.com/gluestack/gluestack-ui/assets/16623517/ac514534-76e2-48e9-ab35-c699cea368f1">

**Comment by Viraj-10** [comment]
> @Viraj-10 This issue I originally reported on this issue was regarding the console warnings on web. That was resolved in 1.1.6 by the PR. There is a regression now where the selected item is not highlighted on the ActionSheet. See my comment and screenshots ☝️
> 
> You example repo is displaying this problem too.

ohh, Got My bad. I thought while pressing it. It's not getting highlighted. 
Since this issue is fixed I closing it.
I have created a new issue here 👉  #1800 . 
Thanks for reporting issue.

**Comment by brianGammon** [comment]
@Viraj-10 Sounds good, thanks!
